tdrs-frontend/src/components/EditProfile/EditProfile.jsx

```diff
@@ -106,13 +119,23 @@ function EditProfile() {
     )
     setErrors(formValidation.errors)
     setTouched(formValidation.touched)
-    setTimeout(() => errorRef.current.focus(), 0)
+
+    if (!Object.values(formValidation.errors).length) {
+      return dispatch(requestAccess(profileInfo))
+    }
+    return setTimeout(() => errorRef.current.focus(), 0)
+  }
+
+  if (requestedAccess && sttAssigned) {
+    return <Redirect to="/request" />
   }
 
   return (
     <div className="grid-container">
-      <h1 className="request-access-header font-serif-2xl">Request Access</h1>
-      <p className="request-access-secondary">
+      <h1 className="font-serif-2xl margin-bottom-0 text-normal">
```

Comment:
I had a custom class that was doing the same thing as these utility classes, so switched over to the utility classes provided by USWDS.